Buildings/Resources/Scripts/Conversion/ConvertBuildings_from_10_to_11.0.0.mos

```diff
@@ -52,3 +52,9 @@ convertClass("Buildings.Experimental.DHC.Plants.Reservoir.Borefield",
              
 // Conversion for https://github.com/lbl-srg/modelica-buildings/issues/3431
 convertClass("Buildings.Fluid.Geothermal.BuriedPipes","Buildings.Fluid.FixedResistances.BuriedPipes");
+
+// Conversion for https://github.com/ibpsa/modelica-ibpsa/issues/1844
+convertClass("Buildings.Airflow.Multizone.BaseClasses.interpolate",
+             "Buildings.Utilities.Math.Functions.interpolate");
+convertClass("Buildings.Airflow.Multizone.BaseClasses.Examples.interpolate",
+             "Buildings.Utilities.Math.Functions.Examples.interpolate");
```

Comment:
Modelica is case sensitive, this should be changed to `Interpolate` for the example.

Reply:
Fixed in e1858c5.